### DISCUSSION TITLE
ProgressBar: set height and color?

### DISCUSSION THREAD

**Discussion by l-o-u-i-s-g** [beeware/toga#2958] [Q&A]
Is there a way to adjust the height  of the  line that displays the ongoing progress ?  The default is very slim and can easily be missed running on Linux on a 2k monitor.  Giving a height with Pack  changes the overall height of the widget  but the line that marks the progression is still very slim.  I tried with font_size  which didn't change anything.   Also  I would have  liked that the  color=  change the color of the progression line.

**Comment by freakboy3742** [comment]
Both of those issues are issues for/with your GTK theme. 

Briefcase is sets the height of the *widget*, but how GTK uses that widget height is up to GTK, and GTK's progress bar is a very thin bar in most themes. If you disagree with that choice, you need to pick a different GTK theme.

As for colors, Toga will make a best-effort attempt to change colors where it can; but Toga ultimately follows the operating system's look and feel for widgets. We might have missed an option to set the color of the bar; but GTK makes it *exceptionally* difficult to change some colors - primarily because apps *shouldn't* be changing the theme colors of apps (a position that, from a philosophical perspective, Toga agrees with).
